frontend_v2/src/app/components/challenge/challengesettings/challengesettings.component.ts

```diff
@@ -934,6 +934,40 @@ export class ChallengesettingsComponent implements OnInit, OnDestroy {
     }
   }
 
+  /**
+   * Edit submission guidelines
+   */
+  editSubmissionGuidelines() {
+    const SELF = this;
+    SELF.apiCall = (params) => {
+      const BODY = JSON.stringify(params);
+      SELF.apiService
+        .patchUrl(SELF.endpointsService.editChallengeDetailsURL(SELF.challenge.creator.id, SELF.challenge.id), BODY)
+        .subscribe(
+          (data) => {
+            SELF.challenge.submission_guidelines = data.submission_guidelines;
+            SELF.globalService.showToast('success', 'The submission guidelines are successfully updated!', 5);
```

Comment:
@gautamjajoo please change the message to `Submission guidelines updated successfully!`

Reply:
Done!